test/cql-pytest/test_secondary_index.py

```diff
@@ -245,3 +245,13 @@ def test_filter_cluster_key(cql, test_keyspace):
         stmt = SimpleStatement(f"SELECT c1, c2 FROM {table} WHERE c1 = 1 and c2 = 1 ALLOW FILTERING")
         rows = cql.execute(stmt)
         assert_rows(rows, [1, 1])
+
+def test_multi_column_with_regular_index(cql, test_keyspace):
+    """Reproduces #9085."""
```

Comment:
Nit: Maybe a comment instead of a string, all other tests have comments

Reply:
Some Pythoners believe that docstrings are better than comments. I think the opposite - while docstrings are good for public API functions, but for tests comments are much better - sometimes you need a comment relevant to multiple tests; comments are indented less so can be fewer lines; comments don't encourage you to fit the comment into one line.

However, I do admit that using docstrings has become almost a religious issue (after all, it is written so in the holy book of PEP-8), so I decided not to fight with people who prefer them :-)